app.js

```diff
@@ -1,13 +1,14 @@
 const express = require("express");
 const {
   getAllTopics,
-  getAllEndpoints
+  getAllEndpoints,
+  getArticleByID
 } = require("./controller/api.controller");
 
 const app = express();
 
 app.get("/api", getAllEndpoints);
 app.get("/api/topics", getAllTopics);
-
+app.get('/api/articles/:article_id', getArticleByID)
 
```

Comment:
I strongly recommend you have another read through the notes for error handling + advanced error handling. You might also want to book in a 1-2-1 with your mentor to talk it through if anything is unclear :)